javacord-core/src/main/java/org/javacord/core/entity/server/ServerImpl.java

```diff
@@ -1407,6 +1407,14 @@ public void selfUndeafen() {
                 this, getConnectedVoiceChannel(api.getYourself()).orElse(null), null, false);
     }
 
+    @Override
+    public CompletableFuture<User> requestMember(long userId) {
+        return new RestRequest<User>(getApi(), RestMethod.GET, RestEndpoint.SERVER_MEMBER)
+                .setUrlParameters(getIdAsString(), Long.toUnsignedString(userId))
+                .execute(result ->  new MemberImpl(api, this, result.getJsonBody(), null).getUser());
+
+    }
```

Comment:
```suggestion
    @Override
    public CompletableFuture<User> requestMember(long userId) {
        return new RestRequest<User>(getApi(), RestMethod.GET, RestEndpoint.SERVER_MEMBER)
                .setUrlParameters(getIdAsString(), Long.toUnsignedString(userId))
                .execute(result -> new MemberImpl(api, this, result.getJsonBody(), null).getUser());
    }
```
Just some nitpicky format changes :)